### DISCUSSION TITLE
LaraDumps

### DISCUSSION THREAD

**Discussion by dansysanalyst** [HassanZahirnia/laravel-package-ocean#188] [Package Suggestions]
# LaraDumps

https://laradumps.dev/

https://github.com/laradumps/laradumps

LaraDumps is a standalone desktop app designed to improve Laravel debugging.

In addition to dumping variables, LaraDumps offers specific dumping for Models, Collection, Str and Mailable classes, along with capture and inspection of SQL Queries, Jobs, Cache, Routes and Artisan commands.

LaraDumps also provides debug tools to validate JSON strings, compare string diffs, display Markdown, elapsed time tracking.

- Free and Open-Source project
- 500+ GitHub stars
- First released in Jun/2022, current version: 3.0.x. 
- Actively maintained with more than 15 contributors.
- Available on Windows, Linux and macOS.
- From the same author of [Livewire PowerGrid](https://github.com/Power-Components/livewire-powergrid) (already featured in Laravel Package Ocean)

## Screenshot

<img src="https://github.com/HassanZahirnia/laravel-package-ocean/assets/79267265/66bfd35e-a66b-4efa-aea6-a848b147fc8b" width="600">

**Comment by HassanZahirnia** [comment]
Thanks for the suggestion 😊 I've added the package!
